redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/testingGeneration.kt

```diff
@@ -287,18 +287,26 @@ internal fun generateWidgetValue(schema: Schema, widget: Widget): FileSpec {
     .addStatement("val widgetTag = %T(%L)", Protocol.WidgetTag, (widget as ProtocolWidget).tag)
     .addStatement(
       """
-      |val childrenDiff = %T(
+      |builder.changes += %T(
+      |  widgetId,
+      |  widgetTag,
+      |)
+      |
+      """.trimMargin(),
+      Protocol.Create,
+    )
+    .addStatement(
+      """
+      |builder.changes += %T(
       |  parentId,
       |  childrenTag,
       |  widgetId,
-      |  widgetTag,
-      |  builder.childrenDiffs.size
+      |  builder.changes.size,
```

Comment:
This previously-broken assumption is now super broken after this PR.

https://github.com/cashapp/redwood/issues/1031